### DISCUSSION TITLE
Display song info

### DISCUSSION THREAD

**Discussion by xxxserxxx** [spezifisch/stmps#29] [Ideas]
Relating to #25.

I'd like to be able to view more metadata about a song; we already get song metadata, we just need a way to display it.

I'm favoring splitting the Queue into two vertical panels, with the currently selected song's metadata displayed in the right column. Initially as simple **key**: value pairs, it could eventually get more fancy with showing cover art, or (lrc) lyrics (I'm not sure if the Subsonic API provides the latter; it definitely provides the former).

An alternative is a hotkey that pops up a help-style dialog wherever a song is selected, on any page.

**Comment by xxxserxxx** [comment]
So, a little more about the motivation for this, which I just noticed again. Often in my queue, I have a bunch of songs (e.g., the entire Rush discography of 20 or so albums) and -- my memory not always being reliable -- I'll want to know which album the song is from. I can't tell from the playing Queue, as stmps displays only the song title. This _could_ be addressed by presenting songs in the Queue as a table with metadata as columns; however, I do think having a separate tab would allow easier viewing of all metadata, and leaves open the option of adding things like cover art and lyrics in the future -- something that would be more difficult if the data is presented in tabular format.

That said, this could also be implemented as a toggle-able presentation format that the user can cycle through to their preference.

**Comment by spezifisch** [comment]
Tracked by #25, I'll add a reference there so I don't forget this thread.
